.github/workflows/ansible-test.yml

```
@@ -5,20 +5,24 @@ on:
 jobs:
   sanity:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python_version: ["3.6"]
+        ansible_version: ["2.9", "2.10"]
     steps:
 
       - name: Check out code
         uses: actions/checkout@v1
         with:
           path: ansible_collections/community/grafana
 
-      - name: Set up Python 3.6
+      - name: Set up Python ${{ matrix.python_version }}
         uses: actions/setup-python@v1
         with:
-          python-version: 3.6
+          python-version: ${{ matrix.python_version }}
 
-      - name: Install ansible-base (devel)
-        run: pip install antsibull-changelog https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check
+      - name: Install ansible
+        run: pip install antsibull-changelog ansible==${{ matrix.ansible_version }}
```

You don't need `antsibull-changelog` for the sanity tests. Also, it's probably better to install ansible-base (and not Ansible) from the latest stable branches, i.e. do
```suggestion
        run: pip install https://github.com/ansible/ansible/archive/${{ matrix.ansible_version }}.tar.gz --disable-pip-version-check
```
and let ansible_version run over `stable-2.9`, `stable-2.10` and `devel`.

---

(You don't need the `antsibull-changelog lint` below - I cannot comment on that line - since `ansible-test sanity` already does that, at least for ansible-test from ansible-base 2.10 and newer.)